src/components/Data/Subscriptions.ts

```diff
@@ -29,3 +34,40 @@ export const SubscriptionData: ISubscription[] = [
     status: "notActive",
   },
 ];
+
+export const SubscriptionTableData: ISubscriptionTabledata[] = [
+  {
+    header: ["Amount", "Type of Plan", "Payment Date", "Card used to pay"],
+
+    data: [
```

Comment:
it should be array of object not direct array.

Reply:
Actually we are using existing table from the storybook, but there is some issue in table component while creation itself we are working on modifying the table component instead of direct array.

Reply:
@Pavankumaryaragogu116 is working on the table component once if it is done i will modify the data accordingly